src/customJs/socialShareTool/SocialShareTool.js

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types';
 
 export const SocialShareTool = (props) => {
   const values = props.values;
+  const { social_share_size: size } = props.values;
+  const { locale } = window['unlayer-extensions-configuration'];
+  const baseUrl = 'https://app2.dopplerfiles.com/MSEditor/images';
+  const sufixLang = locale === 'en' ? '_en' : '';
```

Comment:
These values could be also defined in the locale files like other string literals.

Reply:
Do you mean in the .env file?

Reply:
![image](https://user-images.githubusercontent.com/6733401/209368633-1212dcd0-e65d-494e-8103-58f5473fd158.png)

Reply:
Now we use the lang files ![image](https://user-images.githubusercontent.com/6733401/210255238-ccfea46d-7523-4eec-b4c6-d54e05e85233.png)